install/terraform/modules/gke/cluster.tf

```diff
@@ -142,6 +143,36 @@ resource "google_container_cluster" "primary" {
       }
     }
   }
+  dynamic "windows" {
+    for_each = local.windows ? [] : [1]
+    content {
+      type = "node_pool"
+      node_pool {
+        name       = "windows"
+        node_count = local.initialNodeCount
+        version    = local.kubernetesVersion
+
+        management {
+          auto_upgrade = false
+        }
+
+        node_config {
+          machine_type = local.machineType
```

Comment:
Shouldn't this node pool's config also specify the image-type to be `WINDOWS_LTSC`?